WeakAuras/Transmission.lua

```diff
@@ -667,7 +667,7 @@ local receivedData;
 hooksecurefunc("SetItemRef", function(link, text)
   buttonAnchor:Hide()
   if(link == "garrmission:weakauras") then
-    local _, _, characterName, displayName = text:find("|Hgarrmission:weakauras|h|cFF8800FF%[([^%s]+) |r|cFF8800FF%- ([^%]]+)%]|h");
+    local _, _, characterName, displayName = text:find("|Hgarrmission:weakauras|h|cFF8800FF%[([^%s]+) |r|cFF8800FF%- (.*)%]|h");
```

Comment:
Shouldn't this be a lazy match? .* would likely match all the way up through a second linked aura in the same chat message